### PR TITLE
Fix errors with name of bridge not being given

### PR DESCRIPTION
**Pull request by Amronos**
# 🦟 Bug fix

Fixes errors occurring on running the command `ros2 launch ros_gz_sim ros_gz_spawn_model.launch.py world:=empty file:=$(ros2 pkg prefix --share ros_gz_sim_demos)/models/vehicle/model.sdf name:=my_vehicle x:=5.0 y:=5.0 z:=0.5` given [here](https://gazebosim.org/docs/harmonic/ros2_spawn_model/#spawn-a-model-using-the-launch-file-included-in-ros-gz-sim).
Fixes similar errors occurring with `ros_gz_sim.launch.py`

## Summary
I added a new argument `bridge_name` which is passed to `ros_gz_bridge`'s name argument.
I have also renamed the `name` argument for `spawn_model_description` to `entity_name`.

Before this PR the following error occurred:
`[ERROR] [launch]: Caught exception in launch (see debug for traceback): Included launch description missing required argument 'name' (description: 'Name of ros_gz_bridge node'), given: [config_file, container_name, namespace, use_composition, use_respawn, log_level]`

After the PR the error no longer occurs. The new command should be:
`ros2 launch ros_gz_sim ros_gz_spawn_model.launch.py world:=empty file:=$(ros2 pkg prefix --share ros_gz_sim_demos)/models/vehicle/model.sdf entity_name:=my_vehicle x:=5.0 y:=5.0 z:=0.5 bridge_name:=spawn_model_bridge`

I have made similar changes to `ros_gz_sim.launch.py` due to encountering similar errors with it.

I have also removed `use_composition` argument from `spawn_model_description` as it isn't present in `gz_spawn_model.launch.py`.

I will make a PR for the documentation later, if this one gets approved.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by Amronos** [comment]
I have added `bridge_name` to `ros_gz_sim.launch.py` also due to encountering similar errors with it.

**Comment by ahcorde** [comment]
@caguero do you mind to take a look?

**Review by caguero — CHANGES_REQUESTED** [review]
Thanks for the fix.

Could you also update the following files? There are instances where we could use `bridge_name` and `entity_name` too.
* https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_bridge/launch/ros_gz_bridge.launch
* https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_bridge/launch/ros_gz_bridge.launch.py
* https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py
* https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/launch/gz_spawn_model.launch
* https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_sim/ros_gz_sim/actions/gz_spawn_model.py

**Comment by caguero** [comment]
Mental note: Update the tutorials after merging this PR in https://github.com/gazebosim/docs .

File: ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def __init__(
         :param: log_level Log level.
         """
         super().__init__(**kwargs)
-        self.__name = name
+        self.__bridge_name = bridge_name
```
Comment (caguero) [inline ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py:61]: Nit: Could you update the documentation a few lines above?

File: ros_gz_sim/ros_gz_sim/actions/gz_spawn_model.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def __init__(
         self.__file = file
         self.__xml_string = xml_string
         self.__topic = topic
-        self.__name = name
+        self.__entity_name = entity_name
```
Comment (caguero) [inline ros_gz_sim/ros_gz_sim/actions/gz_spawn_model.py:75]: Nit: Could you update the documentation as well?

**Review by caguero — APPROVED** [review]

File: ros_gz_sim/ros_gz_sim/actions/gz_spawn_model.py (hunk 2 above)
Comment (Amronos) [reply ros_gz_sim/ros_gz_sim/actions/gz_spawn_model.py:75]: done

File: ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py (hunk 1 above)
Comment (Amronos) [reply ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py:61]: done

**Pull request merged**